Formula/sem.rb

```diff
@@ -0,0 +1,16 @@
+class Sem < Formula
+  desc "Command-line tool to manage postgresql database schema migrations"
+  homepage "https://github.com/mbryzek/schema-evolution-manager"
+  url "https://github.com/mbryzek/schema-evolution-manager/archive/0.9.40.tar.gz"
+  sha256 "cf5aab3f2fe81e3daca400811f46a11d4ebd25d7325eca92a83dbe0494e25d5a"
+
+  bottle :unneeded
+
+  def install
+    system "./install.sh", prefix
+  end
+
+  test do
+    assert_equal "0.9.40", shell_output("#{bin}/sem-info version").chomp
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
@fxcoudert added a better test